### DISCUSSION TITLE
Bug: Visual Bug with Chrome 89

### DISCUSSION THREAD

**Discussion by cr3spo9** [gpbl/react-day-picker#1461] [Support]
### Bug description

With Google Chrome 89, an user has reported that his calendar is seen like this:
<img width="972" alt="Screenshot 2022-05-26 at 07 55 32" src="https://user-images.githubusercontent.com/43613261/170474525-3544a06d-625e-4217-8a2a-8dbdd171745b.png">

We have been using react-day-picker for a year and it has seen the first time that we see this, and all the other users are having a good experience with it.

### To reproduce

I have tried with the same version and I could not reproduce it, I dont know if you have had this issue before and know how to fix it.

**Comment by gpbl** [comment]
Definitely not a DayPicker bug: maybe an extension installed? A wrong CSS loaded? Something failing in the build?
